Formula/xidel.rb

```diff
@@ -0,0 +1,60 @@
+class Xidel < Formula
+  desc "XPath/XQuery 3.0, JSONiq interpreter to extract data from HTML/XML/JSON"
+  homepage "https://www.videlibri.de/xidel.html"
+  license "GPL-3.0-or-later"
+
+  stable do
+    url "https://github.com/benibela/xidel/releases/download/Xidel_0.9.8/xidel-0.9.8.src.tar.gz"
+    sha256 "72b5b1a2fc44a0a61831e268c45bc6a6c28e3533b5445151bfbdeaf1562af39c"
+  end
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+    regex(%r{href=.*?/tag/Xidel_(\d+(?:\.\d+)+)["' >]}i)
+  end
+
+  head do
+    url "https://github.com/benibela/xidel.git", branch: "master"
+
+    resource "flre" do
+      url "https://github.com/benibela/flre.git", branch: "master"
+    end
+    resource "internettools" do
+      url "https://github.com/benibela/internettools.git", branch: "master"
+    end
+    resource "pasdblstrutils" do
+      url "https://github.com/BeRo1985/pasdblstrutils.git", branch: "master"
+    end
+    resource "rcmdline" do
+      url "https://github.com/benibela/rcmdline.git", branch: "master"
+    end
+    resource "synapse" do
+      url "http://svn.code.sf.net/p/synalist/code/synapse/40"
+    end
+  end
+
+  depends_on "fpc" => :build
+  depends_on "openssl@1.1"
+
+  def install
+    resources.each do |r|
+      target = buildpath/"import"/r.name
+      mkdir_p target.dirname
+      r.stage target
+    end
+
+    cd File.directory?("programs/internet/xidel") ? "programs/internet/xidel" : "." do
+      inreplace "build.sh" do |s|
+        s.gsub!(/\$fpc /, "$fpc -k-rpath -k#{sh_quote Formula["openssl@1.1"].opt_lib} ")
+      end
```

Comment:
```suggestion
      inreplace "build.sh", "$fpc ", "$fpc -k-rpath -k'#{Formula["openssl@1.1"].opt_lib}' "
```
You don't need a block, a regex, and the need for `sh_quote` is arguable when there is something much simpler.